components/portal/widgets/partials/type__list-of-links.html

```diff
@@ -18,10 +18,24 @@
     under the License.
 
 -->
-<div class="widget-body list-of-links">
+<!-- ZERO LINKS: emulate a basic widget -->
+<div ng-show="config.links.length == 0">
+  <a tabindex="-1"
+    ng-href="{{ widget.url }}"
+    target="{{ widget.target ? widget.target : '_self' }}"
+    rel="noopener noreferrer"
+    class="basic-widget">
+    <div class="widget-icon-container"
+      layout="column" layout-align="center center">
+      <widget-icon></widget-icon>
+    </div>
+  </a>
+</div>
```

Comment:
It'd be nice to refactor this for less duplication between this edge case of the `list-of-links` type `partial` and the embedded `basic` widget implementation in the `widget-card` partial. Maybe extract a `basicWidget` `directive`?